templates/vpc/content/vpc.tf

```diff
@@ -0,0 +1,28 @@
+#
+# VPC
+#
+
+module "vpc" {
+  source                            = "github.com/cds-snc/terraform-modules//vpc?ref=v9.2.5"
+  name                              = ${{ values.vpc_name | dump }}
+  billing_tag_value                 = ${{ values.billing_tag_value| dump }} 
+  availability_zones                = ${{ values.availability_zones | dump }}
+  single_nat_gateway                = ${{ values.single_nat_gateway }}
+  allow_https_requests_in           = ${{ values.allow_https_requests_in | dump }}
+  allow_https_requests_in_response  = ${{ values.allow_https_requests_in_response }}
+  allow_https_requests_out          = ${{ values.allow_https_requests_out | dump }}
+  allow_https_requests_out_response = ${{ values.allow_https_requests_out_response }}
+
+  resource "aws_flow_log" "cloud_based_sensor" {
```

Comment:
Thank you for making all the changes!  Last little tweak is that this `resource` should be outside of the `module` block.

Reply:
Sorry Pat! What a silly mistake for me to make - I think I was on autopilot when I made this change. It has been corrected.